### PR TITLE
added CLI reference content for v1.0.0

### PR DESCRIPTION
**Pull request by zr-msft**

### CODE REVIEW

**Comment by zr-msft** [comment]
@yizha1 @FeynmanZhou @toddysm  can i get a review on this please so i can merge?

**Review by FeynmanZhou — APPROVED** [review]
LGTM

File: content/en/docs/user-guides/cli-reference/notation_verify.md [hunk 1]
```diff
@@ -10,40 +12,37 @@ Verify OCI artifacts
...
-Example - [Experimental] Verify a signature on an OCI artifact referenced in an OCI layout using trust policy statement specified by scope.
-```shell
-notation verify --oci-layout <registry>/<repository>@<digest> --scope <trust_policy_scope>
-```
+### Examples
 
-Example - [Experimental] Verify a signature on an OCI artifact identified by a tag and referenced in an OCI layout using trust policy statement specified by scope.
-```shell
-notation verify --oci-layout <registry>/<repository>:<tag> --scope <trust_policy_scope>
 ```
```
Comment (FeynmanZhou) [inline content/en/docs/user-guides/cli-reference/notation_verify.md:21]: @zr-msft  For experimental features, is there a way to show the examples in CLI reference docs? It seems experimental flags and their examples are not generated by Cobra.

**Review by yizha1 — APPROVED** [review]
LGTM

**Comment by zr-msft** [comment]
@FeynmanZhou  regenerated with experimental flag enabled and manually moved over the changes. PTAL

**Review by yizha1 — APPROVED** [review]
Thanks @zr-msft LGTM

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Pull request merged**